main/ertp/guide/purses-and-payments.md

```diff
@@ -119,38 +117,38 @@ In other words, if you create a `payment` containing
 10 Quatloos or it will be deleted from its `issuer` records and no
 longer have any value. While a `payment` can be either combined with others or
 split into multiple `payments`, in both cases the original `payment(s)`
-is/are deleted and the results put in one or more new `payments`.
+are consumed and the results put in one or more new `payments`.
 
-A `payment` can be deposited in purses, split into multiple 
-`payments`, combined with other `payments`, and claimed (getting an exclusive `payment` and revoking access from anyone else). 
+A `payment` can be deposited into a purse, split into multiple
+`payments`, combined with other `payments`, or claimed (getting an exclusive `payment` and revoking access from anyone else).
```

Comment:
Note that we will probably retire the split and combine operations, and move `claim` from issue to purse. Just noting for you. No change suggested for this PR.